src/components/central-programs-table.js

```diff
@@ -57,7 +57,7 @@ const createFirstRow = (data, totalLabel) => {
 }
 
 const columnsFormatter = (cell, row, rowIndex) => {
-  if (rowIndex === 0) {
+  if (row.name.includes("Total")) {
```

Comment:
genius! much simpler than what I was thinking. Is there a way to pass `totalLabel` here and use that for the check? then if we ever have a 3rd language or if we change that label it will just work?

Reply:
I'm passing totalLabel under `columnsFormatter `